worlds/noita/__init__.py

```diff
@@ -0,0 +1,109 @@
+    burn, freeze, and evaporate your way through the procedurally generated world using wands you've created yourself.
+    """
+    game: str = "Noita"
+    option_definitions = noita_options
+    topology_present = False
+
+    item_name_to_id = item_table
+    location_name_to_id = item_pickups
+
+    data_version = 1
+    forced_auto_forfeit = False
+    web = NoitaWeb()
+
+    def generate_basic(self):
+
+        pool_option = self.world.bad_effects[self.player].value
+        junk_pool: Dict[str, int] = {}
+        junk_pool = item_pool_weights[pool_option].copy()
+        # Generate item pool
+        itempool: List = []
+        for (name, num) in required_items.items():
+            itempool += [name] * num
+        for i in range(1, 1 + self.world.total_locations[self.player].value):
+            itempool += self.world.random.choices(list(junk_pool.keys()), weights=list(junk_pool.values()))
+
+        # Convert itempool into real items
+        itempool = list(map(lambda name: self.create_item(name), itempool))
+        self.world.itempool += itempool
+
+    def create_regions(self) -> None:
+        menu = create_region(self.world, self.player, "Menu")
+        mines = create_region(self.world, self.player, "Mines",
+                                  [f"Chest{i + 1}" for i in range(self.world.total_locations[self.player].value)])
+
+        connection = Entrance(self.player, "Lobby", menu)
+        menu.exits.append(connection)
+        connection.connect(mines)
+
+        self.world.regions += [menu, mines]
+
+        create_events(self.world, self.player)
+
+    def fill_slot_data(self):
+        return {
+            "seed": "".join(self.world.slot_seeds[self.player].choice(string.digits) for i in range(16)),
+            "totalLocations": self.world.total_locations[self.player].value,
+            "badEffects": self.world.bad_effects[self.player].value,
+            "deathLink": self.world.death_link[self.player].value,
+        }
+
+    def create_item(self, name: str) -> Item:
+        item_id = item_table[name]
+        if name == "Heart" or name.startswith("Perk"):
+            classification = ItemClassification.progression
+        elif name.startswith("Wand"):
+            classification = ItemClassification.useful
+        elif name == "Bad":
+            classification = ItemClassification.trap
+        else:
+            classification = ItemClassification.filler
+        item = NoitaItem(name, classification, item_id, self.player)
+        return item
+
+
+def create_events(world: MultiWorld, player: int) -> None:
+    total_locations = world.total_locations[player].value
+    world_region = world.get_region("Mines", player)
+    for i in range(1, 1 + total_locations):
+        event_loc = NoitaLocation(player, f"Pickup{(i + 1)}", None, world_region)
+        event_loc.place_locked_item(NoitaItem(f"Pickup{(i + 1)}", ItemClassification.progression, None, player))
```

Comment:
You already started your range at 1, so this would offset everything by 2. The first "Pickup" event is "Pickup2".

From spoiler:
```
Chest97: Wand (Tier 3)
Chest98: Refresh
Chest99: Perk (Extra Life)
Chest100: Wand (Tier 1)
Pickup2: Pickup2
Pickup3: Pickup3
Pickup4: Pickup4
Pickup5: Pickup5
Pickup6: Pickup6
Pickup7: Pickup7
```